docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,165 @@
+.. _acsincome-data:
+Retiring Adult: ACSIncome
+-------------------------
+
+Introduction
+^^^^^^^^^^^^^^^^^
+
+The ACSIncome dataset is one of five datasets created by Ding et al. [#1]_ 
```

Comment:
I see two [2] in the preview:

![image](https://user-images.githubusercontent.com/24683184/148844966-21989cf1-d1d3-4d55-8bbb-c5d213d45aa1.png)

Reply:
Thanks for pointing this out. I pushed a fix. Once the PR is passing tests and ready to merge, I can squash the temporary fixup commits

Reply:
Don't worry about commits. We're squashing anyway.

Reply:
Is the plan to squash everything into one commit or do we squash into the original 4 commits?

Reply:
We do that upon merging, no need to worry about it on your end.